### PR TITLE
add docs

### PR DESCRIPTION
**Pull request by inftord**
that's it.

### CODE REVIEW

File: docs/api/index.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# API Documentation
+Using the search bar on the left, you may search for certain classes, properties, fields, events or methods. If you are unsure of what you are looking for, you may also use the navigation bar on the left to navigate through the documentation.
+
+If you happen to find any broken links, please open up an issue on the [GitHub Repository](https://github.com/OoLunar/DSharpPlus.CommandAll/issues/new).
```
Comment (OoLunar) [inline docs/api/index.md]: Incorrect repo link

File: docs/docfx.json [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+    "template": [
+      "default",
+      "modern"
+    ],
+    "globalMetadata": {
+      "_appTitle": "DSharpPlus.CommandAll",
+      "_appName": "CommandAll",
+      "_appFaviconPath": "images/favicon.ico",
+      "_appLogoPath": "images/logo.png",
+      "_appFooter": "DSharpPlus.CommandAll"
```
Comment (OoLunar) [inline docs/docfx.json]: Incorrect repo name

File: docs/index.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# DSharpPlus.CommandAll
```
Comment (OoLunar) [inline docs/index.md]: No

**Review by OoLunar — CHANGES_REQUESTED** [review]
You copied and pasted incorrectly

File: docs/index.md (hunk 3 above)
Comment (inftord) [reply docs/index.md]: should i leave index empty then, because im not gonna be the one who is gonna rewrite it

Comment (OoLunar) [reply docs/index.md]: Sure

File: docs/api/index.md (hunk 1 above)
Comment (inftord) [reply docs/api/index.md]: fixed

File: docs/docfx.json (hunk 2 above)
Comment (inftord) [reply docs/docfx.json]: fixed

**Review by OoLunar — APPROVED** [review]
LGTM

**Pull request merged**